src/schema/objects/columns.yaml

```diff
@@ -162,6 +162,34 @@ duration:
   type: number
   unit: s
   minimum: 0
+x_coordinate:
+  name: x_coordinate
+  display_name: Gaze x-coordinate
+  description: |
+    Gaze position x-coordinate of the recorded eye, in the coordinate
+    units specified in the corresponding metadata sidecar.
+  type: number
+y_coordinate:
+  name: y_coordinate
+  display_name: Gaze y-coordinate
+  description: |
+    Gaze position y-coordinate of the recorded eye, in the coordinate
+    units specified in the corresponding metadata sidecar.
+  type: number
+pupil_size:
+  name: pupil_size
+  display_name: Pupil size
+  description: |
+    Pupil size or area of the recorded eye, in the
+    units specified in the corresponding metadata sidecar.
```

Comment:
The description for `pupil_size` says: 

> Pupil size or area of the recorded eye, in the units specified in the corresponding metadata sidecar.

But I don't see any field in `src/schema/objects/metadata.yaml` for specifically specifying the pupil data unit. See my comment there for more info on why I don't think the `SampleCoordinateUnits` currently works for this.

Reply:
Ahh.. I see an example now 🙂 :

```
# sub-01_task-visualSearch_recording-eye1_physio.json

"pupil_size": {
        "Description": "Pupil area of the recorded eye as calculated by the eye-tracker in arbitrary units (see EyeLink's documentation for conversion).",
        "Units": "a.u."
    },
```


Feel free to resolve.